hawkular-alerts-engine/src/test/java/org/hawkular/alerts/engine/PersistenceTest.java

```diff
@@ -2380,4 +2379,108 @@ public void test0140EventTags() throws Exception {
         assertEquals("EventTag2Value", event.getTags().get("EventTag2Name"));
     }
 
+    // These tests would be nice in a separate clas but I couldn't figure how to get multiple test classes
+    // to run embedded Cassandra without stepping on each other.
```

Comment:
Yes, I ran the -Dtest-end=perf many times to execute these tests.   Originally I had introduced two new classes. CassPerfGetTest and PerfGetTest, modeled after CassPersistenceTest and PersistenceTest.   I failed to get these two 'Cass' tests to run together successfully because of issues I had with Cassandra life-cycle.  I feel there is probably a way, but I wasted too much time, and so I merged the new tests into PersistenceTest and removed the new test classes.